### PR TITLE
Update babel-loader and the presets it uses

### PR DESCRIPTION
**Pull request by mmmavis**
Related issue: #1159

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -75,11 +75,11 @@
...
   },
   "dependencies": {
+    "@babel/preset-env": "^7.7.5",
+    "@babel/preset-react": "^7.7.4",
     "autoprefixer": "^9.7.3",
     "babel-core": "^6.26.3",
-    "babel-loader": "^7.1.5",
-    "babel-preset-es2015": "^6.24.1",
-    "babel-preset-react": "^6.24.1",
+    "babel-loader": "^8.0.6",
```
Comment (mmmavis) [inline package.json]: https://babeljs.io/docs/en/env/

File: package.json [hunk 2]
```diff
@@ -39,8 +39,8 @@
   },
   "babel": {
     "presets": [
-      "es2015",
-      "react"
+      "@babel/env",
```
Comment (mmmavis) [inline package.json]: This [part](https://babeljs.io/docs/en/env/#basic-options-docs-en-options-change) in the migration note is incorrect. I got the following error when I `npm start`

```js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: Cannot find module 'babel-preset-env'
- Did you mean "@babel/env"?
```

So I changed the following two lines to `@babel/env` and `@babel/react`

File: package.json (hunk 1 above)
Comment (mmmavis) [inline package.json]: No other breaking changes other than the ones mentioned above.

**Comment by youriwims** [comment]
@mmmavis is this ready to be reviewed?

**Comment by mmmavis** [comment]
@youriwims yup yup!

**Pull request merged**